Common/GPU/ShaderWriter.cpp

```diff
@@ -10,7 +10,6 @@ const char * const vulkan_glsl_preamble_fs =
 "#version 450\n"
 "#extension GL_ARB_separate_shader_objects : enable\n"
 "#extension GL_ARB_shading_language_420pack : enable\n"
-"#extension GL_ARB_conservative_depth : enable\n"
```

Comment:
I thought we needed this for `writer.C("layout (depth_unchanged) out float gl_FragDepth;\n");`?  This is part of that Adreno/Mali bug with discards and stencil.

-[Unknown]

Reply:
That's right, just kept getting compiler warnings about it being unused, but of course should simply check whether it's needed before emitting it, or just ignore it. I'll fix it up when it's time to get this in.